AvaTax.TaxModule.Data/Model/AvaCreateTransactionModel.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System;
+﻿using Avalara.AvaTax.RestClient;
```

Comment:
IMO, `using` directives for `System.*` namespaces should be above all other `using` directives. So, this line should be at line 5, as it was before.

Reply:
How I remember, in our code style documentation write that we should use PowerTools. 
![Powertools](https://user-images.githubusercontent.com/11813599/75341554-c53a4580-58a5-11ea-9fcb-e99c0eae2c9a.png)